File: 1-iniciante/1011.cpp
#include <iostream>
#include <math.h>
#include <cstring>
using namespace std;

int main(){
    double r,v;
    cin>>r;
    v=(4/3.0)*3.14159*pow(r,3);
    cout<<"VOLUME  = ";
    printf("%6.3f%n",v);
    cout<<"\n";
    return 0;
}

File: 1-Iniciante/1011-Esfera-2/1011.cs
using System; 

class URI {

    static void Main(string[] args) { 
        double n;
        
        n=double.Parse(Console.ReadLine());
        double n1=Math.Pow(n, 3);
        Console.WriteLine("VOLUME = {0:0.###}",(4/3.0)*3.14159*n1);
    }

}